### PR TITLE
Eclipse multi language analysis

### PR DESCRIPTION
**Pull request by AZWN**
This PR adds an eclipse plugin that allows language plugins to use multi-language analysis.

### CODE REVIEW

File: example/settings.gradle.kts [hunk 1]
```diff
@@ -60,3 +60,24 @@ fun String.includeProject(id: String, path: String = "$this/$id") {
   includeProject("sdf3.eclipse")
   includeProject("sdf3.intellij")
 }
+
+"multilang/spoofaxcore".run {
```
Comment (Gohla) [inline example/settings.gradle.kts:64]: Maybe I'm missing something, but these projects do not seem to be included in the PR?

**Comment by AZWN** [comment]
Finalized this PR. Can you review it again?
An example can be found at [this branch](https://github.com/AZWN/spoofax-pie/tree/multilang-analysis-example). The `examples\multilang` contains 2 languages with shared analysis. The `multilang.eclipse` module can be used to start an eclipse instance with both plugins, and the `example` module contains an example project that you can import in eclipse.

**Comment by Gohla** [comment]
Superseded by #6
